README.md

```diff
@@ -127,6 +173,43 @@ On the Camera Preview Page, the same second button will toggle between
 the full resolution camera view and a one-to-one pixel view intended
 to aid in focusing the camera (the camera is focused manually).
 
+# Dencam Pages
+
+The Dencam user interface has 5 pages, which are traversed using the 
+top button. Each page is displaying information designated to itself.
+(i.e. Solar Display Page displays information recieved from Sun Saver)
+
+## Networking Information Page
+
+This page displays Raspberry Pi username, wLan0 IP address, and the name of 
+the wifi signal it is connected to.
+
+## Recording Status Page
+
+Displays the current number of video recordings saved and
+stores, the file directory they are currently being stored to, the 
+amount of available storage, time stamp, and a countdown for the 
+recording to start.
+
+## Solar Display Page
+
+This page displays data recieved from the SunSaver decive in realtime.
+Currently it displays Battery Voltage(V), Array Voltage(V),
+Charge current(A), Load current(A), Ah charge(Daily)(Ah), and
+Ah load(Daily)(Ah).
+
+## Camera Preview Page
+
+Displays timestamp and current feed of the Camera, note this feed is 
+not necessarily being recorded.
```

Comment:
should probably explain that second button toggles one-to-one pixel view to aid focusing.